build_tools/cmake/iree_cc_binary.cmake

```diff
@@ -51,7 +51,7 @@ function(iree_cc_binary)
     _RULE
     "HOSTONLY;TESTONLY"
     "NAME"
-    "SRCS;COPTS;DEFINES;LINKOPTS;DATA;DEPS"
+    "SRCS;COPTS;DEFINES;EXCLUDE_FROM_ALL;LINKOPTS;DATA;DEPS"
```

Comment:
Move `EXCLUDE_FROM_ALL` up next to `HOSTONLY` and `TESTONLY`, as it's an "option" (enabled/disabled) and not a "multi value keyword" (arg1;arg2;arg3)

https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html